src/helpers.js

```diff
@@ -100,3 +108,30 @@ export function runQuery (instance, query, needResponse) {
     return response
   })
 }
+
+export function findLastObject (instance, table) {
+  return new Promise((resolve, reject) => {
+    let queryInfo = `PRAGMA table_info("${table}")`
+    let queryId = `SELECT seq FROM sqlite_sequence WHERE name="${table}"`
```

Comment:
I'm sure you already saw it but just in case, ref. https://github.com/citycide/trilogy/issues/21#issuecomment-276739441.